Formula/picotool.rb

```diff
@@ -0,0 +1,30 @@
+class Picotool < Formula
+  desc "RP2040 BOOTSEL and binary utility"
+  homepage "https://github.com/raspberrypi/picotool"
+  url "https://github.com/raspberrypi/picotool/archive/refs/tags/1.1.1.tar.gz"
+  sha256 "2d824dbe48969ab9ae4c5311b15bca3449f5758c43602575c2dc3af13fcba195"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+
+  resource "pico-sdk" do
+    url "https://github.com/raspberrypi/pico-sdk/releases/download/1.5.0/sdk1.5.0-with-submodules.zip"
+    sha256 "59a09e619cab67b614d5f4e928a82f6bd055ce6083eb84f1b6caa269f1e3a559"
+  end
+
+  def install
+    resource("pico-sdk").stage buildpath/"pico-sdk"
+
+    ENV["PICO_SDK_PATH"] = buildpath/"pico-sdk"
+    system "cmake", "."
+    system "cmake", "--build", "."
+
+    bin.install "picotool"
+  end
+
+  test do
+    system "#{bin}/picotool"
```

Comment:
Could you try with a test that covers some of its functionalities? Checks on version or help text almost never fail, even for broken installations. You may check out our [Formula Cookbook](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula) for some useful advice.